rental.rb

```diff
@@ -1,5 +1,6 @@
 class Rental
-  attr_accessor :date, :person, :book
+  attr_accessor :date
+  attr_reader :person, :book
 
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you update the `Getting Started` section in your `README.md` md file by adding correction information to help a user how to proceed if needed. I recommend you refer this **[article](https://gist.github.com/tresorsawasawa/cc5e9a9e5dd20c0179ae98eca3b1ae96)** as example